modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ObjectField/Tabs/BasicInfo/MaxLengthProperties.tsx

```diff
@@ -44,7 +44,10 @@ export function MaxLengthProperties({
 	setValues,
 }: IMaxLengthPropertiesProps) {
 	const [defaultMaxLength, defaultMaxLengthText] =
-		objectField.businessType === 'Text' ? [280, '280'] : [65000, '65,000'];
+		objectField.businessType === 'Text' ||
```

Comment:
@pedro-oliveira446 Just a small change, keep it in alphabetical order like:
`objectField.businessType === 'Encrypted' ||
objectField.businessType === 'Text' ? [280, '280']
			: [65000, '65,000'];`